### ISSUE TITLE
[ARCHETYPE-88] sometimes add-groups fails when there is a pom present

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-archetype#488]
**[Brian E Fox](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=brianf)** opened **[ARCHETYPE-88](https://issues.apache.org/jira/browse/ARCHETYPE-88?redirect=false)** and commented

```
brianf@sonoma /cygdrive/e/svn/Maven/maven-it-tests/maven-integration-test-archetype
$ mvn.bat archetypeng:add-groups -Dgroups=org.apache.maven.something
Using maven.home=c:\Program Files\maven2\bin\\..
[INFO] Scanning for projects...
[INFO] Searching repository for plugin with prefix: 'archetypeng'.
[INFO] ----------------------------------------------------------------------------
[INFO] Building Unnamed - org.apache.maven.its:maven-integration-test-sample-archetype:maven-archetype:1.0-SNAPSHOT
[INFO]    task-segment: [archetypeng:add-groups] (aggregator-style)
[INFO] ----------------------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[ERROR] BUILD ERROR
[INFO] ------------------------------------------------------------------------
[INFO] Internal error in the plugin manager executing goal 'org.apache.maven.plugins:maven-archetypeng-plugin:1.0-SNAPSHOT:add-groups': Unable to find the mojo
'org.apache.maven.plugins:maven-archetypeng-plugin:1.0-SNAPSHOT:add-groups' in the plugin 'org.apache.maven.plugins:maven-archetypeng-plugin'
[INFO] ------------------------------------------------------------------------
[INFO] For more information, run Maven with the -e switch
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 4 seconds
[INFO] Finished at: Tue Aug 21 21:05:49 EDT 2007
[INFO] Final Memory: 6M/13M
[INFO] ------------------------------------------------------------------------
```

---
No further details from [ARCHETYPE-88](https://issues.apache.org/jira/browse/ARCHETYPE-88?redirect=false)

**Comment by jira-importer** [comment]
**[Raphael Luta](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=raphael)** commented

I tested the same command on different projects. some fails, and some succeed.
I also tested other archetypeng goals and they fails or succeed the same.

So what makes this fail or succeed:
a project fails it defines archetypeng as an entension, which is mandatory for using maven-archetype packaging.
